CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.16.6] - 2023-08-20
+### Added
+- "--noprompt" command line argument disabling exit prompt (author: TomKranenburg). See issue #188
```

Comment:
Replace "" with ``

Also your name is no link, replace it with following:

`(author: [TomKranenburg](https://github.com/TomKranenburg)).`

Reply:
Done